### PR TITLE
Check if scene exists

### PR DESCRIPTION
**Pull request by KatekovAnton**
Crash when creating node in memory (not in scene) and setting its tag. Or loading scene from file as prefab. Creating nodes in memory is correct operation afaik and is using to optimize loading process.

### CODE REVIEW

File: Source/Urho3D/Scene/Node.cpp [hunk 1]
```diff
@@ -360,17 +360,16 @@ void Node::AddTag(const String& tag)
 
     // Cache
     if (scene_) {
```
Comment (1vanK) [inline Source/Urho3D/Scene/Node.cpp:362]: use existed code style

Comment (KatekovAnton) [reply Source/Urho3D/Scene/Node.cpp:362]: what?

Comment (1vanK) [reply Source/Urho3D/Scene/Node.cpp:362]: `{` on a separate line

Comment (KatekovAnton) [reply Source/Urho3D/Scene/Node.cpp:362]: fixed!

**Comment by KatekovAnton** [comment]
@1vanK github doesn't show new commits for some reason. closing this pr and creating new to include all changes.

**Pull request closed without merging**